azure-pipelines-dash.yml

```diff
@@ -0,0 +1,86 @@
+# C/C++ with GCC
+# Build your C/C++ project with GCC using make.
+# Add steps that publish test results, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/c-cpp/gcc
+
+pr:
```

Comment:
Does this file need to be merged with the existing `azure-pipelines.yml`? Looks like the automatic PR build will only trigger jobs in `azure-pipelines.yml`

Reply:
Done, please check it.